Update webview examples

nathankot opened the pull request:
* Update wkwebview example to latest swift syntax
* Adds `Allow Arbitrary Loads` to the wkwebview example to circumvent new iOS security defaults
* Removes the selected development team on both webview examples - users should select their own team for signing
* Makes the wkwebview example full-screen with `UINavigationController`

afh commented:
@nathankot What would need to be changed (i.e. served via HTTPS) in order to disallow arbitrary loads?

nathankot commented:
@afh 

> What would need to be changed (i.e. served via HTTPS) in order to disallow arbitrary loads?

Turns out disabling arbitrary loads was only needed because I was using `http` locally :) ⚡️ I've removed that setting in this PR.

@michaelckelly 

I've updated the WKWebView example and removed the UIWebView example leaving a deprecation note.
